### PR TITLE
Pull-Request template file

### PR DESCRIPTION
**Pull request by dcoeurjo**
github has enabled templates in Issues and Pull-Request. 
This PR just adds a new template for pull-requests.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+*Thanks a lot for contributing to DGtal, before submitting your PR, please fill up the description and make sure that all checkboxes are checked.*
```
Comment (kerautret) [inline .github/PULL_REQUEST_TEMPLATE.md]: ALL

**Comment by kerautret** [comment]
greats, did we add :
- [ ] Check the result of [travis](https://travis-ci.org/DGtal-team/DGtal/builds/) and [appveyor](https://ci.appveyor.com/project/dcoeurjo/dgtal-250i1/) continuous-integration tools. 

?
I will add a template to DgtalTools too thanks

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+# PR Description
+
+*your description here*
+
+# Checklist
+
+- [ ] Unit-test of your feature with [Catch](http://dgtal.org/doc/stable/moduleCatch.html).
+- [ ] Doxygen docmentation of the code (classes, methods, types, members...)
```
Comment (rolanddenis) [inline .github/PULL_REQUEST_TEMPLATE.md]: *documentation*

Comment (dcoeurjo) [inline .github/PULL_REQUEST_TEMPLATE.md]: reum... thx.

**Comment by kerautret** [comment]
fine merging

**Pull request merged**